content/applications/websites/website/web_design/building_blocks/features_blocks.rst

```diff
@@ -0,0 +1,28 @@
+===============
+Features blocks
+===============
+
+Use the :guilabel:`Features` blocks to add advanced functionalities to your website, such as
+describing the attributes of a product or service. The blocks from this category allow you to
+present the content of your website in an illustrative method rather than sticking to the textual
+content. This documentation presents the :guilabel:`Table of content` building block.
```

Comment:
This documentation presents the :guilabel:`Table of content` building block.


Presenting it this way it's not typical documentation style. Instead I would say: among different feature blocks, there is the :guilabel:`Table of content` building block --> Explain a little what it does and why we mention it here (one line)